LondonBritishLibrary/orient/BLorient12996.xml

```diff
@@ -0,0 +1,289 @@
+                                    <title ref="STU0009AmharicGram#VerbalNoun">De Abstracto seu Nomine Verbali</title>
+                                    <note>On the verbal noun e.g. the gerund.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.15">
+                                    <locus from="32r"/>
+                                    <title ref="STU0009AmharicGram#TablePronounAffixes">Tabula Pronominum Verbis affixorum</title>
+                                    <note>Table showing the pronominal affixes as they appear on the verb.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.16">
+                                    <locus from="32v"/>
+                                    <title ref="STU0009AmharicGram#TablePresent">Secunda Tabula pro Tempore Praesenti</title>
+                                    <note>Second table shows the verbal forms of the present tense.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.17">
+                                    <locus from="34r"/>
+                                    <title ref="STU0009AmharicGram#Negative">Conjugazio Verbi Negativi, Modo Arabico</title>
+                                    <note>Conjugation of negated verbs in the Arabic mode (sic).</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.18">
+                                    <locus from="35v"/>
+                                    <title ref="STU0009AmharicGram#Dicere">Conjug. Verbi Affirmativi Dicere in Amahara</title>
+                                </msItem>
+                                <msItem xml:id="p1_i1.19">
+                                    <locus from="0000"/>
+                                    <title ref="STU0009AmharicGram#Syntax">Caput ultimum: De Constructione Phrasi</title>
+                                    <note>Final chapter on syntax.</note>
+                                </msItem>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="paper"/>
+                                    </support>
+                                    <extent>
+                                        <measure unit="leaf">37</measure>
+                                        <dimensions type="leaf" unit="mm">
+                                            <height>165</height>
+                                            <width>105</width>
+                                        </dimensions>
+                                    </extent> 
+                                    <foliation>With an original pagination from 1 to 70.</foliation>
```

Comment:
Whose is the foliation? (the question emerges since you refers pagination by do not use it).

Reply:
I assume, that Strelcyn considers it the pagination of the anonymous author. He uses the word "original" also for the title. I will try to rephrase it. 

<img width="434" alt="Screenshot 2024_grammar-original-pagination" src="https://github.com/user-attachments/assets/bd764934-22e0-465f-84e9-12ab1203b7a7">